root/static/scripts/tests/relationship-editor.js

```diff
@@ -158,21 +158,8 @@ function relationshipEditorTest(name, callback) {
                 .append('<div id="content"></div><div id="dialog"></div></div>')
                 .appendTo('body');
 
-        /*
-         * _.defer makes its target functions asynchronous. It is redefined
-         * here to call its target right away, so that we don't have to deal
-         * with writing async tests.
-         */
-        var _defer = _.defer;
-
-        _.defer = function (func) {
-            func.apply(null, Array.from(arguments).slice(1));
-        };
-
```

Comment:
Does setTimeout not need any changes here to replace these?

Reply:
IIRC there was only one test left that needed it, so I just made that test asynchronous (putting the `t.ok` or whatever inside a `setTimeout`).